pkgs/by-name/tr/tracexec/package.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "tracexec";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "kxxt";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-HQTNYReJkfiqS05APtLAUf9dnnPYTzF0ratd5QKv+8w=";
+  };
+
+  checkFlags = [
+    "--skip=cli::test::log_mode_without_args_works" # `Permission denied` (maybe needs root?)
+    "--skip=tracer::test::tracer_emits_exec_event" # needs `/bin/true`
+  ];
+
+  cargoHash = "sha256-Ry7dwaoEAVWl04ug3MI1UX92+ynnnuwA3jfmA/6X+L8=";
+
+  meta = with lib; {
+    platforms = platforms.linux;
+    description = "A small utility for tracing execve{,at} and pre-exec behavior";
+    mainProgram = "tracexec";
+    homepage = "https://github.com/kxxt/tracexec";
+    license = with licenses; [ gpl2 ];
+    maintainers = with maintainers; [ nh2 ];
```

```suggestion
  meta = {
    description = "A small utility for tracing execve{,at} and pre-exec behavior";
    homepage = "https://github.com/kxxt/tracexec";
    license = with lib.licenses; [ gpl2 ];
    mainProgram = "tracexec";
    maintainers = with lib.maintainers; [ nh2 ];
    platforms = platforms.linux;
```
- Using `with` is discouraged, please take a look at https://nix.dev/guides/best-practices#with-scopes and https://github.com/NixOS/nixpkgs/pull/293767
- Order alphabetically